src/lib.rs

```diff
@@ -452,12 +453,19 @@ macro_rules! bitflags {
             }
         }
 
+        impl $crate::__core::iter::Extend<$BitFlags> for $BitFlags {
+            fn extend<T: $crate::__core::iter::IntoIterator<Item=$BitFlags>>(&mut self, iterator: T) {
+                for item in iterator {
+                    self.insert(item)
+                }
+            }
+        }
+
         impl $crate::__core::iter::FromIterator<$BitFlags> for $BitFlags {
+            #[inline]
```

Comment:
It should be fine to remove this `#[inline]` because the function is already generic which will inline it elsewhere anyway